backend/src/main/java/com/woowacourse/momo/participant/domain/Participants.java

```diff
@@ -0,0 +1,81 @@
+package com.woowacourse.momo.participant.domain;
+
+import static com.woowacourse.momo.global.exception.exception.ErrorCode.PARTICIPANT_FINISHED;
+import static com.woowacourse.momo.global.exception.exception.ErrorCode.PARTICIPANT_JOIN_BY_HOST;
+import static com.woowacourse.momo.global.exception.exception.ErrorCode.PARTICIPANT_RE_PARTICIPATE;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.FetchType;
+import javax.persistence.OneToMany;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import com.woowacourse.momo.global.exception.exception.MomoException;
+import com.woowacourse.momo.group.domain.group.Group;
+import com.woowacourse.momo.member.domain.Member;
+
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Embeddable
+public class Participants {
+
+    private static final int NONE_PARTICIPANT = 1;
+
+    @OneToMany(mappedBy = "group", fetch = FetchType.LAZY,
+            cascade = {CascadeType.PERSIST, CascadeType.REMOVE})
+    private final List<Participant> value = new ArrayList<>();
+
+    public Participants(Group group, Member member) {
+        value.add(new Participant(group, member));
+    }
```

Comment:
```suggestion
    public Participants(Group group) {
        value.add(new Participant(group));
    }
```
`member`가 필요한지 의문이에요!!

저번에 말씀주셨을 때의 문제가 아래 `Participant` 생성자 때문이죠?!
```java
    public Participant(Group group, Member member) {
        this.group = group;
        this.member = member;
    }
```
그렇다면 달리 생각해봐서, `Participant` 생성자에 member가 필요할까요?
```java
    public Participant(Group group) {
        this.group = group;
        this.member = group.getHost();
    }
```